src/features/Status/StatusCTAContainer.tsx

```diff
@@ -57,30 +78,94 @@ class StatusCTAContainer extends React.Component<IStatusCTAContainerProps, IStat
+    const { timeZone, country, city } = this.state;
+    newHacker.application.location = {
+      timeZone,
+      country,
+      city,
+    };
+    return newHacker;
+  };
+
   public render() {
+    const { timeZone, country, city } = this.state;
     return (
-      <Flex flex={1} flexDirection="column" justifyContent="center" alignItems="center" p="6px 30px 96px 30px">
+      <Flex
+        flex={1}
+        flexDirection="column"
+        justifyContent="center"
+        alignItems="center"
+        p="6px 30px 96px 30px"
+      >
         {this.props.confirmed && this.props.account ? (
           <StatusCTA
             status={this.state.status}
             firstName={this.props.account.firstName}
             settings={this.state.settings}
-            onClickConfirm={this.confirmStatus}
+            onClickConfirm={async () =>
+              this.setState({ isLocationModalOpen: true })
+            }
             // tslint:disable-next-line: jsx-no-lambda
             onClickWithdraw={() => this.setState({ isModalOpen: true })}
           />
         ) : (
           <ConfirmationEmailSentComponent />
         )}
+        <ConfirmModal
+          isOpen={this.state.isLocationModalOpen}
+          onCanceled={() => this.setState({ isLocationModalOpen: false })}
+          onConfirmed={this.confirmStatus}
+          cancelLabel="Cancel"
+          confirmLabel="Submit"
+        >
+          <Box mb={'10px'} alignSelf={'center'}>
+            <Label style={{ minWidth: '300px', maxWidth: '300px', marginBottom: '1em' }}>
+              What is your time zone?
+              <StyledSelect
+                className="react-select-container"
+                classNamePrefix="react-select"
+                options={TimeZonesLVpair}
+                onChange={this.handleChangeTimeZone}
+                value={{
+                  label: timeZone,
+                  value: timeZone
+                }}
+              />
+            </Label>
+          </Box>
+          <Box mb={'10px'} alignSelf={'center'}>
+            <Label style={{ minWidth: '300px', maxWidth: '300px' }}>
+              In which country are you currently residing?
+              <StyledSelect
+                className="react-select-container"
+                classNamePrefix="react-select"              
+                options={CountriesLVpair}
+                onChange={this.handleChangeCountry}
+                value={{
+                  label: country,
+                  value: country
+                }}
+              />
+            </Label>
+          </Box>
+          <Box mb={'10px'} alignSelf={'center'}>
+            <Label style={{ minWidth: '300px', maxWidth: '300px' }}>
+              In which city are you currently residing?
+              <Input
+                onChange={(e) => this.setState({ city: e.target.value })}
+                value={city}
+              />
+            </Label>
+          </Box>
+        </ConfirmModal>
         <ConfirmModal
```

Comment:
What's the purpose of this modal?

Reply:
one is to collect info on the hacker's current location (which will probably be removed once we get off remote), and the other is a confirmation step before the hacker withdraws

Reply:
Yeah, these fields will be moved to the regular application form and only show up in remote hackathon mode. We didn't realize that we wanted to collect location+timezones info until after apps close so now are collecting them when they confirm